.github/workflows/deploy-helplines-parallel-worflows.yml

```diff
@@ -14,121 +14,121 @@ on:
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
 
   deploy-malawi_staging: 
-    uses: techmatters/serverless/.github/workflows/malawi_staging.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/malawi_staging.yml@master
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
 
   deploy-romania_staging: 
-    uses: techmatters/serverless/.github/workflows/romania_staging.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/romania_staging.yml@master
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
       
 
   deploy-safernet_staging: 
-    uses: techmatters/serverless/.github/workflows/safernet_staging.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/safernet_staging.yml@master
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
      
 
   deploy-south_africa_staging: 
-    uses: techmatters/serverless/.github/workflows/south_africa_staging.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/south_africa_staging.yml@master
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
       
 
   deploy-teguio_colombia_staging: 
-    uses: techmatters/serverless/.github/workflows/teguio_colombia_staging.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/teguio_colombia_staging.yml@master
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
       
   
   deploy-thailand_staging: 
-    uses: techmatters/serverless/.github/workflows/thailand_staging.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/thailand_staging.yml@master
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
       
       
 
   deploy-uk_staging_III: 
-    uses: techmatters/serverless/.github/workflows/uk_staging_III.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/uk_staging_III.yml@master
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
       
       
 
   deploy-uk_staging: 
-    uses: techmatters/serverless/.github/workflows/uk_staging.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/uk_staging.yml@master
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
      
       
 
   deploy-ukraine_staging: 
-    uses: techmatters/serverless/.github/workflows/ukraine_staging.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/ukraine_staging.yml@master
```

Comment:
Can you update this to `hungary_staging.yml` as well?

Reply:
@stephenhand It's done.